nixos/doc/manual/release-notes/rl-1809.xml

```diff
@@ -76,6 +76,12 @@ $ nix-instantiate -E '(import &lt;nixpkgsunstable&gt; {}).gitFull'
   </para>
 
   <itemizedlist>
+   <listitem>
+    <para>
+     The installer is no longer running the graphical session as root.
+     To gain root privileges in the graphical session use <literal>sudo</literal> without a password.
```

Comment:
This should probably be mentioned in the installation section of the manual as well.

Also, it probably should be `sudo -i`.